include/iotc.h

```diff
@@ -357,133 +282,61 @@ extern iotc_state_t iotc_publish_data(iotc_context_handle_t iotc_h,
- *          void* user_data )
- *    </pre>
- *
- * The <code>iotc_user_callback_t</code> parameters are:
- *   - context_handle: The context handle provided to
- *<code>iotc_subscribe</code>.
- *   - call_type: The type of callback invocation. Each callback invocation may
- *be related to a subscription confirmation or a new message.
- *   - params: The details about subscription confirmations or new messages. The
- *data in this structure varies by call_type value.
- *   - state: IOTC_STATE_OK The operation succeeded. For information on other
- *error codes, see the <a href="../../../porting_guide.md">porting guide</a> or
- *<code>examples/</code> directory.
- *   - user_data: The pointer from the corresponding <code>iotc_subscribe</code>
- *user .        parameter.
- *
- * @param [in] iotc_h A context handle created by invoking iotc_create_context.
- * @param [in] topic A string with the name of an MQTT topic.
+ * @brief Subscribes to an MQTT topic.
+ *
+ * @param [in] iotc_h A {@link iotc_create_context() context handle}.
+ * @param [in] topic The MQTT topic.
  * @param [in] qos The Quality of Service (QoS) level. Can be <code>0</code>,
- *<code>1</code>, or <code>2</code>. Cloud IoT Core doesn't support QoS level 2.
- *For more information, see the MQTT specification or
- *<code>iotc_mqtt_qos_e</code> in <code>iotc_mqtt_message.h</code>.
- * @param [in] callback The callback function. Invoked after a message is
- *     successfully or unsuccessfully received.
- * @param [in] user_data (Optional) A pointer which will be passed to the
- *callback function's user_data parameter.
- *
- * @see iotc_create_context
- * @see iotc_connect
- * @see iotc_publish
- * @see iotc_publish_data
- *
- * @retval IOTC_STATE_OK The subscription request is correctly formatted.
- * @retval IOTC_OUT_OF_MEMORY The platform doesn't have enough memory
- *     to fulfull the request.
- * @retval IOTC_INTERNAL_ERROR An unrecoverable error occurred.
+ *     <code>1</code>, or <code>2</code>.
+ * @param [in] callback The {@link ::iotc_user_subscription_callback_t callback}
+ *     invoked after a message is published to the MQTT topic.
+ * @param [in] user_data (Optional) A pointer that to the callback function's
+ *     user_data parameter.
  */
 extern iotc_state_t iotc_subscribe(iotc_context_handle_t iotc_h,
                                    const char* topic, const iotc_mqtt_qos_t qos,
                                    iotc_user_subscription_callback_t* callback,
                                    void* user_data);
 
 /**
- * @brief Asynchronously disconnect from Cloud IoT Core.
+ * @brief Disconnects asynchronously from an MQTT broker.
  *
- * After disconnecting, the Device SDK invokes the <code>iotc_connect()</code>
- * callback with a disconnection status code in the state parameter.
+ * @details After disconnecting, the Device SDK passes the disconnection status
+ * code to iotc_connect(). You may reuse disconnected contexts to reconnect
```

Comment:
...disconnection status to the callback function previously set in the attributes of the corresponding iotc_connect() function call.

Reply:
Done